src/Common/src/Interop/Windows/WebSocket/Interop.WebSocketAbortHandle.cs

```diff
@@ -10,7 +10,6 @@ internal static partial class Interop
     internal static partial class WebSocket
     {
         [DllImport(Libraries.WebSocket)]
-        [SuppressUnmanagedCodeSecurity]
```

Comment:
I don't know which ones get used in OOBs. We only need attributes in files that get used in OOBs (and perhaps ones that would get used in the future).